client/tinychain/graph/app.py

```diff
@@ -75,10 +84,13 @@ class Graph(App):
     a table column which has an edge to itself is an `Edge`, otherwise it's a `ForeignKey`. `ForeignKey` relationships
     are automatically updated when a `Table` is updated, but `Edge` relationships require explicit management.
     """
+    schema = None
+
+    # TODO: remove the and `chain_type` parameters and generate the initial schema via reflection
+    def __init__(self, models: list[Model], chain_type=Sync):
```

Comment:
I think it would be better to leave `schema` as a parameter so that way the calling code has the option of whether or not to auto-generate the schema

Reply:
Can do.

Reply:
https://github.com/haydnv/tinychain/pull/169/commits/3b8d58d692fd4c167eba951051ebc173a17f9ed1